composer.json

```diff
@@ -6,6 +6,9 @@
     "autoload": {
         "psr-4": { "": "src/" }
     },
+    "autoload-dev": {
+        "psr-4": { "Tests\\": "tests/" }
```

Comment:
I don't know... Forcing the namespace to `Tests` instead of an empty one sounds better to me because there should not be any other thing than tests.
 
If we wanted to use `AppBundle\Tests` it would need the PSR-4 to be ` { "" : "test/" } ` and we could put almost anything in it.
 
Plus, the bundle structure is merely optional, even if @fabpot advised adding it.
At first I prepared the structure as `tests/Controller/DefaultControllerTest.php` because here we have the whole application tests, and the bundle structure seemed less important to me than the tests we may add in it.